src/Presentation/UiBuilder.php

```diff
@@ -26,19 +27,27 @@ public function __construct(
 	 * TODO: add integration tests
 	 */
 	public function createHtml( string $searchQuery ): string {
+		$query = $this->parseQuery( $searchQuery );
+		$itemType = $query->getInstanceItemId();
```

Comment:
We have a name mismatch here. Existing code uses `ItemType`, while the new code uses `InstanceItemId` to refer to the same concept.